src/components/Pages/Location/LocationInfo.js

```diff
@@ -26,7 +26,7 @@ const LocationPageContact = ({ phone, email }) => {
   // TODO: use 'libphonenumber-js' for real phone from Joplin API
   const phoneContent = (
     <Fragment>
-      {`${phone.name}: `}
+      {!!phone.name && `${phone.name}: `}
```

Comment:
Fixed a bug where the colon would show if there was no phone 'name'

@Niicck is there any particular reason why we didn't just re-use the Phone component in Contact? We had fixed this bug previously before there.

Reply:
A couple reasons.

1. I didn't have real Joplin data to work with when I wrote it. So the Phone component wouldn't have parsed correctly. I made a note to use `// TODO: use 'libphonenumber-js' for real phone from Joplin API` when the Joplin API got hooked up.
2. The existing Phone component is not a "Phone" component, but a "Phones" component. It has a material icon and different formatting than the design specs for the Location phoneContent section.
  - A valid option would be to abstract out an individual "Phone" component that could be used within both "Phones" and "Location". I did not choose to do that. Instead I wrote 4 lines of code to get the location phone section working and get the mvp shipped. But writing a "Phone" component is a wonderful valid option to improve code consistency and prevent bugs like the one you just fixed.

Reply:
Cool, so we should probably make an issue to do that and evaluate what to do for the components in upcoming events